cmd/kubeadm/app/cmd/phases/join/controlplanejoin.go

```diff
@@ -63,9 +55,10 @@ func getControlPlaneJoinPhaseFlags() []string {
 // NewControlPlaneJoinPhase creates a kubeadm workflow phase that implements joining a machine as a control plane instance
 func NewControlPlaneJoinPhase() workflow.Phase {
 	return workflow.Phase{
-		Name:  "control-plane-join",
-		Short: "Joins a machine as a control plane instance",
-		Long:  cmdutil.MacroCommandLongDescription,
+		Name:    "control-plane-join",
+		Short:   "Joins a machine as a control plane instance",
+		Long:    cmdutil.MacroCommandLongDescription,
+		Example: "kubeadm join phase control-plane-join all",
```

Comment:
please look at similar phase files.
we should keep the `controlPlaneJoinExample` variable for consistency and just add it here:
`Example: controlPlaneJoinExample`

how usefully this is, is another question...but for consistency.